src/libfetchers/cache.cc

```diff
@@ -36,7 +36,7 @@ struct CacheImpl : Cache
     {
         auto state(_state.lock());
 
-        auto dbPath = getCacheDir() + "/nix/fetcher-cache-v2.sqlite";
+        auto dbPath = getCacheDir() + "/fetcher-cache-v3.sqlite";
```

Comment:
@roberth do we want the same version in 2.24 and master?

Reply:
Yes it should be the same version.